### ISSUE TITLE
TypeError: callback is not a function

### ISSUE THREAD

**Issue by bestpika** [chenxsan/vscode-standardjs#465]
**What version of this package are you using?**

- v2.1.2

**What operating system, Node.js, and npm version?**

- Windows | 10 x64
- NodeJS | v16.17.0 x64
- NPM | v8.19.1

**What happened?**

When I try to use the auto-fix feature of this extension, I get the following error message.
```
TypeError: callback is not a function
    at c:\Users\...\.vscode-insiders\extensions\standard.vscode-standard-2.1.2\server\out\server.js:667:17
```

**What did you expect to happen?**

**Are you willing to submit a pull request to fix this bug?**

**Comment by arimgibson** [comment]
Looking into this right now, might be able to submit a PR

Edit: couldn't figure out how to effectively debug the extension with the instructions given in the README, so I'm not going to be able to solve the issue or PR 😞

**Comment by theoludwig** [comment]
Hello, thanks for your report! @bestpika 
I don't have time currently to investigate and fix the issue, but would be happy to review a PR that fixes this.

**Comment by theoludwig** [comment]
> Looking into this right now, might be able to submit a PR
> 
> Edit: couldn't figure out how to effectively debug the extension with the instructions given in the README, so I'm not going to be able to solve the issue or PR disappointed

Hey! @arimgibson 
The instructions to develop the extension are the README: https://github.com/standard/vscode-standard#how-to-develop.
Feel free to post here what are your problems, so we can help you.
Note that, you should first uninstall the extension, before running your "own" develop build of the extension.
Thanks for your interest to contribute.

**Comment by arimgibson** [comment]
Thanks @Divlo! I'm happy to help if I can get the development figured out 😛

I've followed those instructions and have the repo cloned. However, it appears like VSCode/Node can't attach to the server. I've tried with port 6009 (specified in launch.json) on the command line and get `failed to connect, please retry`. Firewall is off, although it shouldn't make a difference because both are running on the same computer

**Comment by theoludwig** [comment]
I tried to reproduce the issue, indeed, there is an error when triggering auto-fix, nevertheless, it seems to work weirdly.
![Peek 2022-09-20 17-16](https://user-images.githubusercontent.com/25207499/191297532-8170b7a1-c7ab-4cad-a11b-fd2ff4321414.gif)

@arimgibson I don't really know what is the issue.
Steps I use to develop the extension:
```sh
git clone git@github.com:standard/vscode-standard.git
cd vscode-standard
npm install
code .
```
Once I'm inside VSCode, I go to the "Run and Debug" tab, and start "Launch Client" task, then another VSCode is opening and I can open any folder, for example, the folder in `client/testFixture/standard` where I can also do `npm install` to install `standard` itself. Hopefully, it helps you to get it working. :smile:
![Screenshot from 2022-09-20 17-10-06](https://user-images.githubusercontent.com/25207499/191298871-c0bbaaa3-5b7c-4743-af72-2982e84b1259.png)

**Comment by arimgibson** [comment]
Hmmm... I'll give the VSCODE debugging another go and will see if I can PR this :)

**Comment by holopekochan** [comment]
The function in the waterfall has been modified to an async function. However, the auto fix features still work with the error with or without async in line https://github.com/standard/vscode-standard/blob/d8b17ac408d0f5fe86ff324ea936acfb961ccbee/server/src/server.ts#L996

async is add-in
commit 3a6d1988cd8e78b817dbdfbbfdefd554a8b3e263

**Comment by josegutierro** [comment]
+1 with this issue

**Comment by theoludwig** [comment]
Thank you everyone for your help!
The issue should be resolved since [v2.1.3](https://github.com/standard/vscode-standard/releases/tag/v2.1.3) that just released.
Please reopen an issue if the issue persists. :smile:
